hedera-node/build.gradle.kts

```diff
@@ -20,6 +20,11 @@ plugins {
 
 description = "Hedera Services Node"
 
+subprojects {
+    project(":hedera-node:hedera-app")
+    project(":hedera-node:hedera-app-api")
+}
+
```

Comment:
I don't think you need to put this here, I think the lines in settings.gradle.kts are enough?

Reply:
Gradle is so smart. Fixed